### ISSUE TITLE
autogen.sh: error: could not find libtool.  libtool is required to run autogen.sh

### ISSUE THREAD

**Issue by ggitau** [zeromq/libzmq#1385]
I get the above error on Ubuntu 15.04 despite having installed libtool. I realised that the libtool command is not available but the libtoolize command is .I was able to build normally by changing the autogen.sh line below
```
command -v libtool >/dev/null 2>&1
```
to
```
command -v libtoolize >/dev/null 2>&1
```

**Comment by youssefhassan** [comment]
Thanks for pointing out the issue.

I did this and it worked also
ln -s /usr/bin/libtoolize /usr/bin/libtool

**Comment by sanzhars** [comment]
I got the same error in Ubuntu 14.04 while trying to install JZMQ. 
Could you please tell how do I write above commands in 14.04?

**Comment by sanzhars** [comment]
Somehow I need to install libtool-bin and now I can run ./autogen.sh . But now I can't run make and sudo make install. I get "make: *** No targets specified and no makefile found.  Stop." error.

**Comment by jonaslejon** [comment]
Thanks. Having the same issue

**Comment by bluca** [comment]
The libtool executable script in Debian and Ubuntu has been moved to a different package, libtool-bin, since it is not actually needed (autotool chain will generate it on the fly). This was fixes in PR #1497 so this issue can be closed.
